docs/topics/ref_resolved-issues-7-2-0.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.2.0/master.adoc
+
+:_template-generated: 2024-12-04
+:_mod-docs-content-type: REFERENCE
+
+[id="resolved-issues-7-2-0_{context}"]
+= Resolved issues
+
+This section provides highlighted issues that have been resolved in {ProductFullName} 7.2.0.
+
+.{ProductShortName} azure-appservice target is missing
+
+In previous versions of {ProductShortName}, the `azure-appservice` target was not available in the CLI distribution of {ProductFullName}. This issue has been resolved in the {ProductShortName} CLI. link:https://issues.redhat.com/browse/MTA-4036[(MTA-4036)]
```

Comment:
A suggestion:  
Remove "In previous versions of MTA...This issue has been resolved in the MTA 7.2.0"
I think it can be understood simply from the title "Resolved issues"

Reply:
No